frontend/apps/web/components/providers/account-provider.tsx

```diff
@@ -58,7 +58,7 @@ export default function AccountProvider(props: Props): ReactElement {
     const foundAccount = accountsResponse?.accounts.find(
       (a) => a.name === accountName
     );
-    if (userAccount?.id === foundAccount?.id) {
+    if (userAccount && foundAccount && userAccount?.id === foundAccount?.id) {
```

Comment:
pesky little fucker.

hitting an edge case where `userAccount?.id === foudnAccout?.id` were both undefined which made it equal true.